src/unix/pty.cc

```diff
@@ -203,7 +203,11 @@ NAN_METHOD(PtyFork) {
 
   // termios
   struct termios* term = new termios();
+#if defined(IUTF8)
   term->c_iflag = ICRNL | IXON | IXANY | IMAXBEL | BRKINT | IUTF8;
+#else
```

Comment:
Actually couldn't. On FreeBSD, `UTF8` isn't picked up as defined, so compile errors. I did not research the whys... Does seem strange.

If you still think it should, I can research if there is a define missing...